python/ct/crypto/cert.py

```diff
@@ -221,6 +221,14 @@ def get_extensions(self):
         """
         return self._asn1_cert["tbsCertificate"]["extensions"] or []
 
+    def get_tbscertificate(self):
```

Comment:
Python getters don't typically start with "get_" (the `get_extensions` method above is an anomaly, even within this class).